cpp/README.md

```diff
@@ -0,0 +1,57 @@
+gRPC CPP Wallet example
+=======================
+
+Bazel
+
+Bazel is the primary build system used by the core gRPC development team. Bazel
```

Comment:
I don't think we need to explain those for this example. Just the instruction to build and run should be enough.

Reply:
done